.Rbuildignore

```diff
@@ -0,0 +1,4 @@
+^doc$
+^Meta$
+^\.github$
+^codecov\.yml$
```

Comment:
maybe add pictures/ media/ maybe idea/ and codecov.yml?

Reply:
hmm, and pictures too? some of them are used is readme... or it doesn't mind?

Reply:
yeah README doesn't get to the R package. (afaik this is a list of stuff that is _not_ needed when installing the package in R)